src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../../types';
+
+import { findPersonByName } from '../../utils/findPersonByName';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+export const PersonInfo: FC<Props> = ({ person, people }) => {
+  const { personSlug } = useParams();
+
+  const mother = findPersonByName(people, person.motherName);
+  const father = findPersonByName(people, person.fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': person.slug === personSlug })}
+    >
+      <td>
+        <Link
```

Comment:
[TASK] create the < PersonLink person={person} /  > and use it for all existing people;